pnpm-workspace.yaml

```diff
@@ -39,3 +39,5 @@ packageImportMethod: clone-or-copy
 patchedDependencies:
   # Needed after the upgrade to TypeScript 5.9.2.
   '@types/wicg-file-system-access@2023.10.6': web/patches/@types__wicg-file-system-access@2023.10.6.patch
+  # Needed for roundRect support which was merged to jest-canvas-mock but not released.
```

Comment:
Can we add a `TODO(ryanclark)` so we don't forget about this?

Reply:
I just checked it quickly by changing package.json to install jest-canvas-mock@2.5.1. `pnpm install` will error if a patch is unused. So we can leave it like this and we'll catch it during dep updates.

```
$ pnpm i
Scope: all 7 workspace projects
 ERR_PNPM_UNUSED_PATCH  The following patches were not used: jest-canvas-mock@2.5.2

Either remove them from "patchedDependencies" or update them to match packages in your dependencies.
Progress: resolved 1518, reused 1446, downloaded 1, added 0
$ echo $?
1
```